lib/vsc/install/commontest.py

```diff
@@ -36,10 +36,10 @@
 
 import optparse
 import os
-import pkg_resources
 import pprint
 import re
 import sys
+import pkg_resources
```

Comment:
Any reason not to keep this alphabetical?

Reply:
pylint says the order is wrong

Reply:
why?

Reply:
dependencies between the modules I assume.

Reply:
fair enough.